jaxrs-api/pom.xml

```diff
@@ -585,6 +585,12 @@
             <version>4.11</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
As already mentioned in #604, I'm fine with adding Mockito. But I agree that we should wait for #604 to be merged before merging this one. This way the history will be much cleaner. I know this defers the merge of this PR, but I think it is worth the effort.

Reply:
I merged #604 meanwhile.